src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderBooking.kt

```diff
@@ -89,6 +91,14 @@ interface BookingDsl {
     dsl: CSIPReportDsl.() -> Unit = {},
   ): CSIPReport
 
+  @IWPDocumentDslMarker
```

Comment:
IWP documents can exist with or without a booking
- have added here for those with a booking